BSP/Inc/BSP_SPI.h

```diff
@@ -47,7 +47,7 @@ void BSP_SPI_SetClock(spi_speed_t speed);
  * @param   txLen   length of data array.
  * @return  None
  */
-void BSP_SPI_Write(spi_port_t port, uint8_t *txBuf, uint32_t txLen);
+bool BSP_SPI_Write(spi_port_t port, uint8_t *txBuf, uint32_t txLen);
```

Comment:
Need to change the comments for the return value. Also, if this is an error, make it an `ErrorStatus` type.

Reply:
Replace all bool returns with `ErrorStatus`